src/cascadia/TerminalSettingsModel/defaults-universal.json

```diff
@@ -78,14 +106,21 @@
         { "command": "closeWindow", "keys": "alt+f4" },
         { "command": "toggleFullscreen", "keys": "alt+enter" },
         { "command": "toggleFullscreen", "keys": "f11" },
+        { "command": "toggleFocusMode" },
+        { "command": "toggleAlwaysOnTop" },

```

Comment:
Oh man will these work in universal?

Reply:
nope, but there's also no point in leaving them off. Many things act _weird_ in WTU, but we should undiverge where we can :)